From: =?UTF-8?q?J=C3=A1n=20Tomko?= <jtomko@redhat.com>
Date: Thu, 12 May 2016 14:19:52 +0200
Subject: [PATCH] Fix tests to include video ram size

My commit 3e42867 started filling out the video size in post-parse,
but did not adjust the tests.

(cherry picked from commit 96b21fb0ecf8242ceb298607da61b5718511a388)
---
 tests/sexpr2xmldata/sexpr2xml-curmem.xml                            | 2 +-
 tests/sexpr2xmldata/sexpr2xml-fv-autoport.xml                       | 2 +-
 tests/sexpr2xmldata/sexpr2xml-fv-empty-kernel.xml                   | 2 +-
 tests/sexpr2xmldata/sexpr2xml-fv-force-hpet.xml                     | 2 +-
 tests/sexpr2xmldata/sexpr2xml-fv-force-nohpet.xml                   | 2 +-
 tests/sexpr2xmldata/sexpr2xml-fv-localtime.xml                      | 2 +-
 tests/sexpr2xmldata/sexpr2xml-fv-net-netfront.xml                   | 2 +-
 tests/sexpr2xmldata/sexpr2xml-fv-parallel-tcp.xml                   | 2 +-
 tests/sexpr2xmldata/sexpr2xml-fv-serial-dev-2-ports.xml             | 2 +-
 tests/sexpr2xmldata/sexpr2xml-fv-serial-dev-2nd-port.xml            | 2 +-
 tests/sexpr2xmldata/sexpr2xml-fv-serial-file.xml                    | 2 +-
 tests/sexpr2xmldata/sexpr2xml-fv-serial-null.xml                    | 2 +-
 tests/sexpr2xmldata/sexpr2xml-fv-serial-pipe.xml                    | 2 +-
 tests/sexpr2xmldata/sexpr2xml-fv-serial-pty.xml                     | 2 +-
 tests/sexpr2xmldata/sexpr2xml-fv-serial-stdio.xml                   | 2 +-
 tests/sexpr2xmldata/sexpr2xml-fv-serial-tcp-telnet.xml              | 2 +-
 tests/sexpr2xmldata/sexpr2xml-fv-serial-tcp.xml                     | 2 +-
 tests/sexpr2xmldata/sexpr2xml-fv-serial-udp.xml                     | 2 +-
 tests/sexpr2xmldata/sexpr2xml-fv-serial-unix.xml                    | 2 +-
 tests/sexpr2xmldata/sexpr2xml-fv-sound-all.xml                      | 2 +-
 tests/sexpr2xmldata/sexpr2xml-fv-sound.xml                          | 2 +-
 tests/sexpr2xmldata/sexpr2xml-fv-usbmouse.xml                       | 2 +-
 tests/sexpr2xmldata/sexpr2xml-fv-usbtablet.xml                      | 2 +-
 tests/sexpr2xmldata/sexpr2xml-fv-utc.xml                            | 2 +-
 tests/sexpr2xmldata/sexpr2xml-fv-v2.xml                             | 2 +-
 tests/sexpr2xmldata/sexpr2xml-fv.xml                                | 2 +-
 tests/sexpr2xmldata/sexpr2xml-no-source-cdrom.xml                   | 2 +-
 tests/sexpr2xmldata/sexpr2xml-pv-vfb-new-vncdisplay.xml             | 2 +-
 tests/sexpr2xmldata/sexpr2xml-pv-vfb-new.xml                        | 2 +-
 tests/sexpr2xmldata/sexpr2xml-pv-vfb-type-crash.xml                 | 2 +-
 tests/sexpr2xmldata/sexpr2xml-vif-rate.xml                          | 2 +-
 tests/xlconfigdata/test-disk-positional-parms-full.xml              | 2 +-
 tests/xlconfigdata/test-disk-positional-parms-partial.xml           | 2 +-
 tests/xlconfigdata/test-fullvirt-direct-kernel-boot-bogus-extra.xml | 2 +-
 tests/xlconfigdata/test-fullvirt-direct-kernel-boot-extra.xml       | 2 +-
 tests/xlconfigdata/test-fullvirt-direct-kernel-boot.xml             | 2 +-
 tests/xlconfigdata/test-fullvirt-multiusb.xml                       | 2 +-
 tests/xlconfigdata/test-fullvirt-nohap.xml                          | 2 +-
 tests/xlconfigdata/test-new-disk.xml                                | 2 +-
 tests/xlconfigdata/test-rbd-multihost-noauth.xml                    | 2 +-
 tests/xlconfigdata/test-spice-features.xml                          | 2 +-
 tests/xlconfigdata/test-spice.xml                                   | 2 +-
 tests/xlconfigdata/test-vif-rate.xml                                | 2 +-
 tests/xmconfigdata/test-escape-paths.xml                            | 2 +-
 tests/xmconfigdata/test-fullvirt-default-feature.xml                | 2 +-
 tests/xmconfigdata/test-fullvirt-force-hpet.xml                     | 2 +-
 tests/xmconfigdata/test-fullvirt-force-nohpet.xml                   | 2 +-
 tests/xmconfigdata/test-fullvirt-localtime.xml                      | 2 +-
 tests/xmconfigdata/test-fullvirt-net-netfront.xml                   | 2 +-
 tests/xmconfigdata/test-fullvirt-new-cdrom.xml                      | 2 +-
 tests/xmconfigdata/test-fullvirt-nohap.xml                          | 2 +-
 tests/xmconfigdata/test-fullvirt-parallel-tcp.xml                   | 2 +-
 tests/xmconfigdata/test-fullvirt-serial-dev-2-ports.xml             | 2 +-
 tests/xmconfigdata/test-fullvirt-serial-dev-2nd-port.xml            | 2 +-
 tests/xmconfigdata/test-fullvirt-serial-file.xml                    | 2 +-
 tests/xmconfigdata/test-fullvirt-serial-null.xml                    | 2 +-
 tests/xmconfigdata/test-fullvirt-serial-pipe.xml                    | 2 +-
 tests/xmconfigdata/test-fullvirt-serial-pty.xml                     | 2 +-
 tests/xmconfigdata/test-fullvirt-serial-stdio.xml                   | 2 +-
 tests/xmconfigdata/test-fullvirt-serial-tcp-telnet.xml              | 2 +-
 tests/xmconfigdata/test-fullvirt-serial-tcp.xml                     | 2 +-
 tests/xmconfigdata/test-fullvirt-serial-udp.xml                     | 2 +-
 tests/xmconfigdata/test-fullvirt-serial-unix.xml                    | 2 +-
 tests/xmconfigdata/test-fullvirt-sound.xml                          | 2 +-
 tests/xmconfigdata/test-fullvirt-usbmouse.xml                       | 2 +-
 tests/xmconfigdata/test-fullvirt-usbtablet.xml                      | 2 +-
 tests/xmconfigdata/test-fullvirt-utc.xml                            | 2 +-
 tests/xmconfigdata/test-no-source-cdrom.xml                         | 2 +-
 tests/xmconfigdata/test-paravirt-net-e1000.xml                      | 2 +-
 tests/xmconfigdata/test-paravirt-net-vifname.xml                    | 2 +-
 tests/xmconfigdata/test-paravirt-new-pvfb-vncdisplay.xml            | 2 +-
 tests/xmconfigdata/test-paravirt-new-pvfb.xml                       | 2 +-
 tests/xmconfigdata/test-pci-devs.xml                                | 2 +-
 73 files changed, 73 insertions(+), 73 deletions(-)

diff --git a/tests/sexpr2xmldata/sexpr2xml-curmem.xml b/tests/sexpr2xmldata/sexpr2xml-curmem.xml
index a2668d1..6932e33 100644
--- a/tests/sexpr2xmldata/sexpr2xml-curmem.xml
+++ b/tests/sexpr2xmldata/sexpr2xml-curmem.xml
@@ -35,7 +35,7 @@
     <input type='keyboard' bus='xen'/>
     <graphics type='vnc' port='-1' autoport='yes'/>
     <video>
-      <model type='xen' heads='1' primary='yes'/>
+      <model type='xen' vram='4096' heads='1' primary='yes'/>
     </video>
     <memballoon model='xen'/>
   </devices>
diff --git a/tests/sexpr2xmldata/sexpr2xml-fv-autoport.xml b/tests/sexpr2xmldata/sexpr2xml-fv-autoport.xml
index c4b22f6..fdd7c7d 100644
--- a/tests/sexpr2xmldata/sexpr2xml-fv-autoport.xml
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-autoport.xml
@@ -51,7 +51,7 @@
     <input type='keyboard' bus='ps2'/>
     <graphics type='vnc' port='5925' autoport='yes' keymap='en-us'/>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='16384' heads='1' primary='yes'/>
     </video>
     <memballoon model='xen'/>
   </devices>
diff --git a/tests/sexpr2xmldata/sexpr2xml-fv-empty-kernel.xml b/tests/sexpr2xmldata/sexpr2xml-fv-empty-kernel.xml
index 17d4016..42d529c 100644
--- a/tests/sexpr2xmldata/sexpr2xml-fv-empty-kernel.xml
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-empty-kernel.xml
@@ -44,7 +44,7 @@
     <input type='keyboard' bus='ps2'/>
     <graphics type='vnc' port='-1' autoport='yes' keymap='ja'/>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='16384' heads='1' primary='yes'/>
     </video>
     <memballoon model='xen'/>
   </devices>
diff --git a/tests/sexpr2xmldata/sexpr2xml-fv-force-hpet.xml b/tests/sexpr2xmldata/sexpr2xml-fv-force-hpet.xml
index 20a0d76..5e26804 100644
--- a/tests/sexpr2xmldata/sexpr2xml-fv-force-hpet.xml
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-force-hpet.xml
@@ -47,7 +47,7 @@
     <input type='keyboard' bus='ps2'/>
     <graphics type='vnc' port='-1' autoport='yes'/>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='16384' heads='1' primary='yes'/>
     </video>
     <memballoon model='xen'/>
   </devices>
diff --git a/tests/sexpr2xmldata/sexpr2xml-fv-force-nohpet.xml b/tests/sexpr2xmldata/sexpr2xml-fv-force-nohpet.xml
index 13fb299..f6af578 100644
--- a/tests/sexpr2xmldata/sexpr2xml-fv-force-nohpet.xml
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-force-nohpet.xml
@@ -47,7 +47,7 @@
     <input type='keyboard' bus='ps2'/>
     <graphics type='vnc' port='-1' autoport='yes'/>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='16384' heads='1' primary='yes'/>
     </video>
     <memballoon model='xen'/>
   </devices>
diff --git a/tests/sexpr2xmldata/sexpr2xml-fv-localtime.xml b/tests/sexpr2xmldata/sexpr2xml-fv-localtime.xml
index 1380426..aba6045 100644
--- a/tests/sexpr2xmldata/sexpr2xml-fv-localtime.xml
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-localtime.xml
@@ -44,7 +44,7 @@
     <input type='keyboard' bus='ps2'/>
     <graphics type='vnc' port='-1' autoport='yes' keymap='ja'/>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='16384' heads='1' primary='yes'/>
     </video>
     <memballoon model='xen'/>
   </devices>
diff --git a/tests/sexpr2xmldata/sexpr2xml-fv-net-netfront.xml b/tests/sexpr2xmldata/sexpr2xml-fv-net-netfront.xml
index 64111fe..22e2035 100644
--- a/tests/sexpr2xmldata/sexpr2xml-fv-net-netfront.xml
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-net-netfront.xml
@@ -45,7 +45,7 @@
     <input type='keyboard' bus='ps2'/>
     <graphics type='vnc' port='-1' autoport='yes' keymap='ja'/>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='16384' heads='1' primary='yes'/>
     </video>
     <memballoon model='xen'/>
   </devices>
diff --git a/tests/sexpr2xmldata/sexpr2xml-fv-parallel-tcp.xml b/tests/sexpr2xmldata/sexpr2xml-fv-parallel-tcp.xml
index f362aee..aa5122b 100644
--- a/tests/sexpr2xmldata/sexpr2xml-fv-parallel-tcp.xml
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-parallel-tcp.xml
@@ -49,7 +49,7 @@
     <input type='keyboard' bus='ps2'/>
     <graphics type='vnc' port='-1' autoport='yes'/>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='16384' heads='1' primary='yes'/>
     </video>
     <memballoon model='xen'/>
   </devices>
diff --git a/tests/sexpr2xmldata/sexpr2xml-fv-serial-dev-2-ports.xml b/tests/sexpr2xmldata/sexpr2xml-fv-serial-dev-2-ports.xml
index 4d3bdde..bcccb94 100644
--- a/tests/sexpr2xmldata/sexpr2xml-fv-serial-dev-2-ports.xml
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-serial-dev-2-ports.xml
@@ -56,7 +56,7 @@
     <input type='keyboard' bus='ps2'/>
     <graphics type='vnc' port='-1' autoport='yes'/>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='16384' heads='1' primary='yes'/>
     </video>
     <memballoon model='xen'/>
   </devices>
diff --git a/tests/sexpr2xmldata/sexpr2xml-fv-serial-dev-2nd-port.xml b/tests/sexpr2xmldata/sexpr2xml-fv-serial-dev-2nd-port.xml
index d3eaf82..82c41a6 100644
--- a/tests/sexpr2xmldata/sexpr2xml-fv-serial-dev-2nd-port.xml
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-serial-dev-2nd-port.xml
@@ -52,7 +52,7 @@
     <input type='keyboard' bus='ps2'/>
     <graphics type='vnc' port='-1' autoport='yes'/>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='16384' heads='1' primary='yes'/>
     </video>
     <memballoon model='xen'/>
   </devices>
diff --git a/tests/sexpr2xmldata/sexpr2xml-fv-serial-file.xml b/tests/sexpr2xmldata/sexpr2xml-fv-serial-file.xml
index 0f209d6..062f349 100644
--- a/tests/sexpr2xmldata/sexpr2xml-fv-serial-file.xml
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-serial-file.xml
@@ -52,7 +52,7 @@
     <input type='keyboard' bus='ps2'/>
     <graphics type='vnc' port='-1' autoport='yes'/>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='16384' heads='1' primary='yes'/>
     </video>
     <memballoon model='xen'/>
   </devices>
diff --git a/tests/sexpr2xmldata/sexpr2xml-fv-serial-null.xml b/tests/sexpr2xmldata/sexpr2xml-fv-serial-null.xml
index b157f63..38a2fb3 100644
--- a/tests/sexpr2xmldata/sexpr2xml-fv-serial-null.xml
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-serial-null.xml
@@ -50,7 +50,7 @@
     <input type='keyboard' bus='ps2'/>
     <graphics type='vnc' port='-1' autoport='yes'/>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='16384' heads='1' primary='yes'/>
     </video>
     <memballoon model='xen'/>
   </devices>
diff --git a/tests/sexpr2xmldata/sexpr2xml-fv-serial-pipe.xml b/tests/sexpr2xmldata/sexpr2xml-fv-serial-pipe.xml
index d9cd840..4d76e34 100644
--- a/tests/sexpr2xmldata/sexpr2xml-fv-serial-pipe.xml
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-serial-pipe.xml
@@ -52,7 +52,7 @@
     <input type='keyboard' bus='ps2'/>
     <graphics type='vnc' port='-1' autoport='yes'/>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='16384' heads='1' primary='yes'/>
     </video>
     <memballoon model='xen'/>
   </devices>
diff --git a/tests/sexpr2xmldata/sexpr2xml-fv-serial-pty.xml b/tests/sexpr2xmldata/sexpr2xml-fv-serial-pty.xml
index 2cffa65..e0f90ef 100644
--- a/tests/sexpr2xmldata/sexpr2xml-fv-serial-pty.xml
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-serial-pty.xml
@@ -50,7 +50,7 @@
     <input type='keyboard' bus='ps2'/>
     <graphics type='vnc' port='-1' autoport='yes'/>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='16384' heads='1' primary='yes'/>
     </video>
     <memballoon model='xen'/>
   </devices>
diff --git a/tests/sexpr2xmldata/sexpr2xml-fv-serial-stdio.xml b/tests/sexpr2xmldata/sexpr2xml-fv-serial-stdio.xml
index 6e473d4..7de5731 100644
--- a/tests/sexpr2xmldata/sexpr2xml-fv-serial-stdio.xml
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-serial-stdio.xml
@@ -50,7 +50,7 @@
     <input type='keyboard' bus='ps2'/>
     <graphics type='vnc' port='-1' autoport='yes'/>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='16384' heads='1' primary='yes'/>
     </video>
     <memballoon model='xen'/>
   </devices>
diff --git a/tests/sexpr2xmldata/sexpr2xml-fv-serial-tcp-telnet.xml b/tests/sexpr2xmldata/sexpr2xml-fv-serial-tcp-telnet.xml
index f03bd29..c22e2a5 100644
--- a/tests/sexpr2xmldata/sexpr2xml-fv-serial-tcp-telnet.xml
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-serial-tcp-telnet.xml
@@ -54,7 +54,7 @@
     <input type='keyboard' bus='ps2'/>
     <graphics type='vnc' port='-1' autoport='yes'/>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='16384' heads='1' primary='yes'/>
     </video>
     <memballoon model='xen'/>
   </devices>
diff --git a/tests/sexpr2xmldata/sexpr2xml-fv-serial-tcp.xml b/tests/sexpr2xmldata/sexpr2xml-fv-serial-tcp.xml
index 2e9c629..439a66e 100644
--- a/tests/sexpr2xmldata/sexpr2xml-fv-serial-tcp.xml
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-serial-tcp.xml
@@ -54,7 +54,7 @@
     <input type='keyboard' bus='ps2'/>
     <graphics type='vnc' port='-1' autoport='yes'/>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='16384' heads='1' primary='yes'/>
     </video>
     <memballoon model='xen'/>
   </devices>
diff --git a/tests/sexpr2xmldata/sexpr2xml-fv-serial-udp.xml b/tests/sexpr2xmldata/sexpr2xml-fv-serial-udp.xml
index 3f3f676..b4bdc8a 100644
--- a/tests/sexpr2xmldata/sexpr2xml-fv-serial-udp.xml
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-serial-udp.xml
@@ -54,7 +54,7 @@
     <input type='keyboard' bus='ps2'/>
     <graphics type='vnc' port='-1' autoport='yes'/>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='16384' heads='1' primary='yes'/>
     </video>
     <memballoon model='xen'/>
   </devices>
diff --git a/tests/sexpr2xmldata/sexpr2xml-fv-serial-unix.xml b/tests/sexpr2xmldata/sexpr2xml-fv-serial-unix.xml
index 4f28511..af9f928 100644
--- a/tests/sexpr2xmldata/sexpr2xml-fv-serial-unix.xml
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-serial-unix.xml
@@ -52,7 +52,7 @@
     <input type='keyboard' bus='ps2'/>
     <graphics type='vnc' port='-1' autoport='yes'/>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='16384' heads='1' primary='yes'/>
     </video>
     <memballoon model='xen'/>
   </devices>
diff --git a/tests/sexpr2xmldata/sexpr2xml-fv-sound-all.xml b/tests/sexpr2xmldata/sexpr2xml-fv-sound-all.xml
index f934a85..d0240c4 100644
--- a/tests/sexpr2xmldata/sexpr2xml-fv-sound-all.xml
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-sound-all.xml
@@ -46,7 +46,7 @@
     <sound model='sb16'/>
     <sound model='es1370'/>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='16384' heads='1' primary='yes'/>
     </video>
     <memballoon model='xen'/>
   </devices>
diff --git a/tests/sexpr2xmldata/sexpr2xml-fv-sound.xml b/tests/sexpr2xmldata/sexpr2xml-fv-sound.xml
index f934a85..d0240c4 100644
--- a/tests/sexpr2xmldata/sexpr2xml-fv-sound.xml
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-sound.xml
@@ -46,7 +46,7 @@
     <sound model='sb16'/>
     <sound model='es1370'/>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='16384' heads='1' primary='yes'/>
     </video>
     <memballoon model='xen'/>
   </devices>
diff --git a/tests/sexpr2xmldata/sexpr2xml-fv-usbmouse.xml b/tests/sexpr2xmldata/sexpr2xml-fv-usbmouse.xml
index 8b5887e..e84e172 100644
--- a/tests/sexpr2xmldata/sexpr2xml-fv-usbmouse.xml
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-usbmouse.xml
@@ -45,7 +45,7 @@
     <input type='keyboard' bus='ps2'/>
     <graphics type='vnc' port='-1' autoport='yes' keymap='ja'/>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='16384' heads='1' primary='yes'/>
     </video>
     <memballoon model='xen'/>
   </devices>
diff --git a/tests/sexpr2xmldata/sexpr2xml-fv-usbtablet.xml b/tests/sexpr2xmldata/sexpr2xml-fv-usbtablet.xml
index 3e4b366..187d867 100644
--- a/tests/sexpr2xmldata/sexpr2xml-fv-usbtablet.xml
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-usbtablet.xml
@@ -45,7 +45,7 @@
     <input type='keyboard' bus='ps2'/>
     <graphics type='vnc' port='-1' autoport='yes' keymap='ja'/>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='16384' heads='1' primary='yes'/>
     </video>
     <memballoon model='xen'/>
   </devices>
diff --git a/tests/sexpr2xmldata/sexpr2xml-fv-utc.xml b/tests/sexpr2xmldata/sexpr2xml-fv-utc.xml
index 9c2b9e3..2816f4a 100644
--- a/tests/sexpr2xmldata/sexpr2xml-fv-utc.xml
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-utc.xml
@@ -44,7 +44,7 @@
     <input type='keyboard' bus='ps2'/>
     <graphics type='vnc' port='-1' autoport='yes' keymap='ja'/>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='16384' heads='1' primary='yes'/>
     </video>
     <memballoon model='xen'/>
   </devices>
diff --git a/tests/sexpr2xmldata/sexpr2xml-fv-v2.xml b/tests/sexpr2xmldata/sexpr2xml-fv-v2.xml
index 9c2b9e3..2816f4a 100644
--- a/tests/sexpr2xmldata/sexpr2xml-fv-v2.xml
+++ b/tests/sexpr2xmldata/sexpr2xml-fv-v2.xml
@@ -44,7 +44,7 @@
     <input type='keyboard' bus='ps2'/>
     <graphics type='vnc' port='-1' autoport='yes' keymap='ja'/>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='16384' heads='1' primary='yes'/>
     </video>
     <memballoon model='xen'/>
   </devices>
diff --git a/tests/sexpr2xmldata/sexpr2xml-fv.xml b/tests/sexpr2xmldata/sexpr2xml-fv.xml
index 9c2b9e3..2816f4a 100644
--- a/tests/sexpr2xmldata/sexpr2xml-fv.xml
+++ b/tests/sexpr2xmldata/sexpr2xml-fv.xml
@@ -44,7 +44,7 @@
     <input type='keyboard' bus='ps2'/>
     <graphics type='vnc' port='-1' autoport='yes' keymap='ja'/>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='16384' heads='1' primary='yes'/>
     </video>
     <memballoon model='xen'/>
   </devices>
diff --git a/tests/sexpr2xmldata/sexpr2xml-no-source-cdrom.xml b/tests/sexpr2xmldata/sexpr2xml-no-source-cdrom.xml
index 97d9482..41fb3b7 100644
--- a/tests/sexpr2xmldata/sexpr2xml-no-source-cdrom.xml
+++ b/tests/sexpr2xmldata/sexpr2xml-no-source-cdrom.xml
@@ -49,7 +49,7 @@
     <input type='keyboard' bus='ps2'/>
     <graphics type='vnc' port='-1' autoport='yes'/>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='16384' heads='1' primary='yes'/>
     </video>
     <memballoon model='xen'/>
   </devices>
diff --git a/tests/sexpr2xmldata/sexpr2xml-pv-vfb-new-vncdisplay.xml b/tests/sexpr2xmldata/sexpr2xml-pv-vfb-new-vncdisplay.xml
index dbdacf9..72fec25 100644
--- a/tests/sexpr2xmldata/sexpr2xml-pv-vfb-new-vncdisplay.xml
+++ b/tests/sexpr2xmldata/sexpr2xml-pv-vfb-new-vncdisplay.xml
@@ -30,7 +30,7 @@
       <listen type='address' address='0.0.0.0'/>
     </graphics>
     <video>
-      <model type='xen' heads='1' primary='yes'/>
+      <model type='xen' vram='4096' heads='1' primary='yes'/>
     </video>
     <memballoon model='xen'/>
   </devices>
diff --git a/tests/sexpr2xmldata/sexpr2xml-pv-vfb-new.xml b/tests/sexpr2xmldata/sexpr2xml-pv-vfb-new.xml
index c415aab..0fd7a88 100644
--- a/tests/sexpr2xmldata/sexpr2xml-pv-vfb-new.xml
+++ b/tests/sexpr2xmldata/sexpr2xml-pv-vfb-new.xml
@@ -30,7 +30,7 @@
       <listen type='address' address='0.0.0.0'/>
     </graphics>
     <video>
-      <model type='xen' heads='1' primary='yes'/>
+      <model type='xen' vram='4096' heads='1' primary='yes'/>
     </video>
     <memballoon model='xen'/>
   </devices>
diff --git a/tests/sexpr2xmldata/sexpr2xml-pv-vfb-type-crash.xml b/tests/sexpr2xmldata/sexpr2xml-pv-vfb-type-crash.xml
index c1acba1..ee24536 100644
--- a/tests/sexpr2xmldata/sexpr2xml-pv-vfb-type-crash.xml
+++ b/tests/sexpr2xmldata/sexpr2xml-pv-vfb-type-crash.xml
@@ -33,7 +33,7 @@
     <input type='keyboard' bus='xen'/>
     <graphics type='vnc' port='-1' autoport='yes'/>
     <video>
-      <model type='xen' heads='1' primary='yes'/>
+      <model type='xen' vram='4096' heads='1' primary='yes'/>
     </video>
     <memballoon model='xen'/>
   </devices>
diff --git a/tests/sexpr2xmldata/sexpr2xml-vif-rate.xml b/tests/sexpr2xmldata/sexpr2xml-vif-rate.xml
index 45887e5..1ebd5f0 100644
--- a/tests/sexpr2xmldata/sexpr2xml-vif-rate.xml
+++ b/tests/sexpr2xmldata/sexpr2xml-vif-rate.xml
@@ -48,7 +48,7 @@
     <input type='keyboard' bus='ps2'/>
     <graphics type='vnc' port='-1' autoport='yes' keymap='ja'/>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='16384' heads='1' primary='yes'/>
     </video>
     <memballoon model='xen'/>
   </devices>
diff --git a/tests/xlconfigdata/test-disk-positional-parms-full.xml b/tests/xlconfigdata/test-disk-positional-parms-full.xml
index 3d4868c..41e8804 100644
--- a/tests/xlconfigdata/test-disk-positional-parms-full.xml
+++ b/tests/xlconfigdata/test-disk-positional-parms-full.xml
@@ -52,7 +52,7 @@
       <listen type='address' address='127.0.0.1'/>
     </graphics>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='8192' heads='1' primary='yes'/>
     </video>
   </devices>
 </domain>
diff --git a/tests/xlconfigdata/test-disk-positional-parms-partial.xml b/tests/xlconfigdata/test-disk-positional-parms-partial.xml
index 402f479..6578e59 100644
--- a/tests/xlconfigdata/test-disk-positional-parms-partial.xml
+++ b/tests/xlconfigdata/test-disk-positional-parms-partial.xml
@@ -52,7 +52,7 @@
       <listen type='address' address='127.0.0.1'/>
     </graphics>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='8192' heads='1' primary='yes'/>
     </video>
   </devices>
 </domain>
diff --git a/tests/xlconfigdata/test-fullvirt-direct-kernel-boot-bogus-extra.xml b/tests/xlconfigdata/test-fullvirt-direct-kernel-boot-bogus-extra.xml
index 686a409..3738c8e 100644
--- a/tests/xlconfigdata/test-fullvirt-direct-kernel-boot-bogus-extra.xml
+++ b/tests/xlconfigdata/test-fullvirt-direct-kernel-boot-bogus-extra.xml
@@ -49,7 +49,7 @@
       <listen type='address' address='127.0.0.1'/>
     </graphics>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='8192' heads='1' primary='yes'/>
     </video>
   </devices>
 </domain>
diff --git a/tests/xlconfigdata/test-fullvirt-direct-kernel-boot-extra.xml b/tests/xlconfigdata/test-fullvirt-direct-kernel-boot-extra.xml
index 686a409..3738c8e 100644
--- a/tests/xlconfigdata/test-fullvirt-direct-kernel-boot-extra.xml
+++ b/tests/xlconfigdata/test-fullvirt-direct-kernel-boot-extra.xml
@@ -49,7 +49,7 @@
       <listen type='address' address='127.0.0.1'/>
     </graphics>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='8192' heads='1' primary='yes'/>
     </video>
   </devices>
 </domain>
diff --git a/tests/xlconfigdata/test-fullvirt-direct-kernel-boot.xml b/tests/xlconfigdata/test-fullvirt-direct-kernel-boot.xml
index 686a409..3738c8e 100644
--- a/tests/xlconfigdata/test-fullvirt-direct-kernel-boot.xml
+++ b/tests/xlconfigdata/test-fullvirt-direct-kernel-boot.xml
@@ -49,7 +49,7 @@
       <listen type='address' address='127.0.0.1'/>
     </graphics>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='8192' heads='1' primary='yes'/>
     </video>
   </devices>
 </domain>
diff --git a/tests/xlconfigdata/test-fullvirt-multiusb.xml b/tests/xlconfigdata/test-fullvirt-multiusb.xml
index 5ec72c4..d7df23a 100644
--- a/tests/xlconfigdata/test-fullvirt-multiusb.xml
+++ b/tests/xlconfigdata/test-fullvirt-multiusb.xml
@@ -48,7 +48,7 @@
       <listen type='address' address='127.0.0.1'/>
     </graphics>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='8192' heads='1' primary='yes'/>
     </video>
   </devices>
 </domain>
diff --git a/tests/xlconfigdata/test-fullvirt-nohap.xml b/tests/xlconfigdata/test-fullvirt-nohap.xml
index a997c7a..9cd7b0b 100644
--- a/tests/xlconfigdata/test-fullvirt-nohap.xml
+++ b/tests/xlconfigdata/test-fullvirt-nohap.xml
@@ -56,7 +56,7 @@
       <listen type='address' address='127.0.0.1'/>
     </graphics>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='8192' heads='1' primary='yes'/>
     </video>
   </devices>
 </domain>
diff --git a/tests/xlconfigdata/test-new-disk.xml b/tests/xlconfigdata/test-new-disk.xml
index 3d4868c..41e8804 100644
--- a/tests/xlconfigdata/test-new-disk.xml
+++ b/tests/xlconfigdata/test-new-disk.xml
@@ -52,7 +52,7 @@
       <listen type='address' address='127.0.0.1'/>
     </graphics>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='8192' heads='1' primary='yes'/>
     </video>
   </devices>
 </domain>
diff --git a/tests/xlconfigdata/test-rbd-multihost-noauth.xml b/tests/xlconfigdata/test-rbd-multihost-noauth.xml
index f8f663f..728aa1e 100644
--- a/tests/xlconfigdata/test-rbd-multihost-noauth.xml
+++ b/tests/xlconfigdata/test-rbd-multihost-noauth.xml
@@ -49,7 +49,7 @@
       <listen type='address' address='127.0.0.1'/>
     </graphics>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='8192' heads='1' primary='yes'/>
     </video>
   </devices>
 </domain>
diff --git a/tests/xlconfigdata/test-spice-features.xml b/tests/xlconfigdata/test-spice-features.xml
index f9eb857..3820732 100644
--- a/tests/xlconfigdata/test-spice-features.xml
+++ b/tests/xlconfigdata/test-spice-features.xml
@@ -48,7 +48,7 @@
       <clipboard copypaste='yes'/>
     </graphics>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='8192' heads='1' primary='yes'/>
     </video>
   </devices>
 </domain>
diff --git a/tests/xlconfigdata/test-spice.xml b/tests/xlconfigdata/test-spice.xml
index fc2fa3f..f33691f 100644
--- a/tests/xlconfigdata/test-spice.xml
+++ b/tests/xlconfigdata/test-spice.xml
@@ -48,7 +48,7 @@
       <clipboard copypaste='no'/>
     </graphics>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='8192' heads='1' primary='yes'/>
     </video>
   </devices>
 </domain>
diff --git a/tests/xlconfigdata/test-vif-rate.xml b/tests/xlconfigdata/test-vif-rate.xml
index 0e13f2e..3620e2a 100644
--- a/tests/xlconfigdata/test-vif-rate.xml
+++ b/tests/xlconfigdata/test-vif-rate.xml
@@ -55,7 +55,7 @@
       <listen type='address' address='127.0.0.1'/>
     </graphics>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='8192' heads='1' primary='yes'/>
     </video>
   </devices>
 </domain>
diff --git a/tests/xmconfigdata/test-escape-paths.xml b/tests/xmconfigdata/test-escape-paths.xml
index 6373fdf..a5daa2c 100644
--- a/tests/xmconfigdata/test-escape-paths.xml
+++ b/tests/xmconfigdata/test-escape-paths.xml
@@ -54,7 +54,7 @@
     <sound model='sb16'/>
     <sound model='es1370'/>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='16384' heads='1' primary='yes'/>
     </video>
     <memballoon model='xen'/>
   </devices>
diff --git a/tests/xmconfigdata/test-fullvirt-default-feature.xml b/tests/xmconfigdata/test-fullvirt-default-feature.xml
index 6f4dae6..ce1280d 100644
--- a/tests/xmconfigdata/test-fullvirt-default-feature.xml
+++ b/tests/xmconfigdata/test-fullvirt-default-feature.xml
@@ -48,7 +48,7 @@
       <listen type='address' address='127.0.0.1'/>
     </graphics>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='16384' heads='1' primary='yes'/>
     </video>
     <memballoon model='xen'/>
   </devices>
diff --git a/tests/xmconfigdata/test-fullvirt-force-hpet.xml b/tests/xmconfigdata/test-fullvirt-force-hpet.xml
index 6f4dae6..ce1280d 100644
--- a/tests/xmconfigdata/test-fullvirt-force-hpet.xml
+++ b/tests/xmconfigdata/test-fullvirt-force-hpet.xml
@@ -48,7 +48,7 @@
       <listen type='address' address='127.0.0.1'/>
     </graphics>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='16384' heads='1' primary='yes'/>
     </video>
     <memballoon model='xen'/>
   </devices>
diff --git a/tests/xmconfigdata/test-fullvirt-force-nohpet.xml b/tests/xmconfigdata/test-fullvirt-force-nohpet.xml
index 6015405..7636f31 100644
--- a/tests/xmconfigdata/test-fullvirt-force-nohpet.xml
+++ b/tests/xmconfigdata/test-fullvirt-force-nohpet.xml
@@ -48,7 +48,7 @@
       <listen type='address' address='127.0.0.1'/>
     </graphics>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='16384' heads='1' primary='yes'/>
     </video>
     <memballoon model='xen'/>
   </devices>
diff --git a/tests/xmconfigdata/test-fullvirt-localtime.xml b/tests/xmconfigdata/test-fullvirt-localtime.xml
index 5f81e25..b3adf86 100644
--- a/tests/xmconfigdata/test-fullvirt-localtime.xml
+++ b/tests/xmconfigdata/test-fullvirt-localtime.xml
@@ -46,7 +46,7 @@
       <listen type='address' address='127.0.0.1'/>
     </graphics>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='16384' heads='1' primary='yes'/>
     </video>
     <memballoon model='xen'/>
   </devices>
diff --git a/tests/xmconfigdata/test-fullvirt-net-netfront.xml b/tests/xmconfigdata/test-fullvirt-net-netfront.xml
index cb0615b..c22ad8d 100644
--- a/tests/xmconfigdata/test-fullvirt-net-netfront.xml
+++ b/tests/xmconfigdata/test-fullvirt-net-netfront.xml
@@ -46,7 +46,7 @@
       <listen type='address' address='127.0.0.1'/>
     </graphics>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='16384' heads='1' primary='yes'/>
     </video>
     <memballoon model='xen'/>
   </devices>
diff --git a/tests/xmconfigdata/test-fullvirt-new-cdrom.xml b/tests/xmconfigdata/test-fullvirt-new-cdrom.xml
index d29892e..2bb3d72 100644
--- a/tests/xmconfigdata/test-fullvirt-new-cdrom.xml
+++ b/tests/xmconfigdata/test-fullvirt-new-cdrom.xml
@@ -46,7 +46,7 @@
       <listen type='address' address='127.0.0.1'/>
     </graphics>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='16384' heads='1' primary='yes'/>
     </video>
     <memballoon model='xen'/>
   </devices>
diff --git a/tests/xmconfigdata/test-fullvirt-nohap.xml b/tests/xmconfigdata/test-fullvirt-nohap.xml
index f38366c..234856c 100644
--- a/tests/xmconfigdata/test-fullvirt-nohap.xml
+++ b/tests/xmconfigdata/test-fullvirt-nohap.xml
@@ -47,7 +47,7 @@
       <listen type='address' address='127.0.0.1'/>
     </graphics>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='16384' heads='1' primary='yes'/>
     </video>
     <memballoon model='xen'/>
   </devices>
diff --git a/tests/xmconfigdata/test-fullvirt-parallel-tcp.xml b/tests/xmconfigdata/test-fullvirt-parallel-tcp.xml
index 3269aee..be149fd 100644
--- a/tests/xmconfigdata/test-fullvirt-parallel-tcp.xml
+++ b/tests/xmconfigdata/test-fullvirt-parallel-tcp.xml
@@ -51,7 +51,7 @@
       <listen type='address' address='127.0.0.1'/>
     </graphics>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='16384' heads='1' primary='yes'/>
     </video>
     <memballoon model='xen'/>
   </devices>
diff --git a/tests/xmconfigdata/test-fullvirt-serial-dev-2-ports.xml b/tests/xmconfigdata/test-fullvirt-serial-dev-2-ports.xml
index a20d6c8..753043d 100644
--- a/tests/xmconfigdata/test-fullvirt-serial-dev-2-ports.xml
+++ b/tests/xmconfigdata/test-fullvirt-serial-dev-2-ports.xml
@@ -58,7 +58,7 @@
       <listen type='address' address='127.0.0.1'/>
     </graphics>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='16384' heads='1' primary='yes'/>
     </video>
     <memballoon model='xen'/>
   </devices>
diff --git a/tests/xmconfigdata/test-fullvirt-serial-dev-2nd-port.xml b/tests/xmconfigdata/test-fullvirt-serial-dev-2nd-port.xml
index 8344da3..a104c6f 100644
--- a/tests/xmconfigdata/test-fullvirt-serial-dev-2nd-port.xml
+++ b/tests/xmconfigdata/test-fullvirt-serial-dev-2nd-port.xml
@@ -54,7 +54,7 @@
       <listen type='address' address='127.0.0.1'/>
     </graphics>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='16384' heads='1' primary='yes'/>
     </video>
     <memballoon model='xen'/>
   </devices>
diff --git a/tests/xmconfigdata/test-fullvirt-serial-file.xml b/tests/xmconfigdata/test-fullvirt-serial-file.xml
index b9439d7..f27aba9 100644
--- a/tests/xmconfigdata/test-fullvirt-serial-file.xml
+++ b/tests/xmconfigdata/test-fullvirt-serial-file.xml
@@ -54,7 +54,7 @@
       <listen type='address' address='127.0.0.1'/>
     </graphics>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='16384' heads='1' primary='yes'/>
     </video>
     <memballoon model='xen'/>
   </devices>
diff --git a/tests/xmconfigdata/test-fullvirt-serial-null.xml b/tests/xmconfigdata/test-fullvirt-serial-null.xml
index 6e8196b..ea29944 100644
--- a/tests/xmconfigdata/test-fullvirt-serial-null.xml
+++ b/tests/xmconfigdata/test-fullvirt-serial-null.xml
@@ -52,7 +52,7 @@
       <listen type='address' address='127.0.0.1'/>
     </graphics>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='16384' heads='1' primary='yes'/>
     </video>
     <memballoon model='xen'/>
   </devices>
diff --git a/tests/xmconfigdata/test-fullvirt-serial-pipe.xml b/tests/xmconfigdata/test-fullvirt-serial-pipe.xml
index da0c9bd..69bf51c 100644
--- a/tests/xmconfigdata/test-fullvirt-serial-pipe.xml
+++ b/tests/xmconfigdata/test-fullvirt-serial-pipe.xml
@@ -54,7 +54,7 @@
       <listen type='address' address='127.0.0.1'/>
     </graphics>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='16384' heads='1' primary='yes'/>
     </video>
     <memballoon model='xen'/>
   </devices>
diff --git a/tests/xmconfigdata/test-fullvirt-serial-pty.xml b/tests/xmconfigdata/test-fullvirt-serial-pty.xml
index acff588..0c6e477 100644
--- a/tests/xmconfigdata/test-fullvirt-serial-pty.xml
+++ b/tests/xmconfigdata/test-fullvirt-serial-pty.xml
@@ -52,7 +52,7 @@
       <listen type='address' address='127.0.0.1'/>
     </graphics>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='16384' heads='1' primary='yes'/>
     </video>
     <memballoon model='xen'/>
   </devices>
diff --git a/tests/xmconfigdata/test-fullvirt-serial-stdio.xml b/tests/xmconfigdata/test-fullvirt-serial-stdio.xml
index 6909053..1f12081 100644
--- a/tests/xmconfigdata/test-fullvirt-serial-stdio.xml
+++ b/tests/xmconfigdata/test-fullvirt-serial-stdio.xml
@@ -52,7 +52,7 @@
       <listen type='address' address='127.0.0.1'/>
     </graphics>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='16384' heads='1' primary='yes'/>
     </video>
     <memballoon model='xen'/>
   </devices>
diff --git a/tests/xmconfigdata/test-fullvirt-serial-tcp-telnet.xml b/tests/xmconfigdata/test-fullvirt-serial-tcp-telnet.xml
index a3fd449..47544df 100644
--- a/tests/xmconfigdata/test-fullvirt-serial-tcp-telnet.xml
+++ b/tests/xmconfigdata/test-fullvirt-serial-tcp-telnet.xml
@@ -56,7 +56,7 @@
       <listen type='address' address='127.0.0.1'/>
     </graphics>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='16384' heads='1' primary='yes'/>
     </video>
     <memballoon model='xen'/>
   </devices>
diff --git a/tests/xmconfigdata/test-fullvirt-serial-tcp.xml b/tests/xmconfigdata/test-fullvirt-serial-tcp.xml
index 9dba76d..8f1777a 100644
--- a/tests/xmconfigdata/test-fullvirt-serial-tcp.xml
+++ b/tests/xmconfigdata/test-fullvirt-serial-tcp.xml
@@ -56,7 +56,7 @@
       <listen type='address' address='127.0.0.1'/>
     </graphics>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='16384' heads='1' primary='yes'/>
     </video>
     <memballoon model='xen'/>
   </devices>
diff --git a/tests/xmconfigdata/test-fullvirt-serial-udp.xml b/tests/xmconfigdata/test-fullvirt-serial-udp.xml
index 3577f92..c59e2b6 100644
--- a/tests/xmconfigdata/test-fullvirt-serial-udp.xml
+++ b/tests/xmconfigdata/test-fullvirt-serial-udp.xml
@@ -56,7 +56,7 @@
       <listen type='address' address='127.0.0.1'/>
     </graphics>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='16384' heads='1' primary='yes'/>
     </video>
     <memballoon model='xen'/>
   </devices>
diff --git a/tests/xmconfigdata/test-fullvirt-serial-unix.xml b/tests/xmconfigdata/test-fullvirt-serial-unix.xml
index 919108b..e678786 100644
--- a/tests/xmconfigdata/test-fullvirt-serial-unix.xml
+++ b/tests/xmconfigdata/test-fullvirt-serial-unix.xml
@@ -54,7 +54,7 @@
       <listen type='address' address='127.0.0.1'/>
     </graphics>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='16384' heads='1' primary='yes'/>
     </video>
     <memballoon model='xen'/>
   </devices>
diff --git a/tests/xmconfigdata/test-fullvirt-sound.xml b/tests/xmconfigdata/test-fullvirt-sound.xml
index 0e85953..3bd2e76 100644
--- a/tests/xmconfigdata/test-fullvirt-sound.xml
+++ b/tests/xmconfigdata/test-fullvirt-sound.xml
@@ -48,7 +48,7 @@
     <sound model='sb16'/>
     <sound model='es1370'/>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='16384' heads='1' primary='yes'/>
     </video>
     <memballoon model='xen'/>
   </devices>
diff --git a/tests/xmconfigdata/test-fullvirt-usbmouse.xml b/tests/xmconfigdata/test-fullvirt-usbmouse.xml
index 8eb5aa5..f1421b6 100644
--- a/tests/xmconfigdata/test-fullvirt-usbmouse.xml
+++ b/tests/xmconfigdata/test-fullvirt-usbmouse.xml
@@ -47,7 +47,7 @@
       <listen type='address' address='127.0.0.1'/>
     </graphics>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='16384' heads='1' primary='yes'/>
     </video>
     <memballoon model='xen'/>
   </devices>
diff --git a/tests/xmconfigdata/test-fullvirt-usbtablet.xml b/tests/xmconfigdata/test-fullvirt-usbtablet.xml
index dcdd951..566c842 100644
--- a/tests/xmconfigdata/test-fullvirt-usbtablet.xml
+++ b/tests/xmconfigdata/test-fullvirt-usbtablet.xml
@@ -47,7 +47,7 @@
       <listen type='address' address='127.0.0.1'/>
     </graphics>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='16384' heads='1' primary='yes'/>
     </video>
     <memballoon model='xen'/>
   </devices>
diff --git a/tests/xmconfigdata/test-fullvirt-utc.xml b/tests/xmconfigdata/test-fullvirt-utc.xml
index d29892e..2bb3d72 100644
--- a/tests/xmconfigdata/test-fullvirt-utc.xml
+++ b/tests/xmconfigdata/test-fullvirt-utc.xml
@@ -46,7 +46,7 @@
       <listen type='address' address='127.0.0.1'/>
     </graphics>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='16384' heads='1' primary='yes'/>
     </video>
     <memballoon model='xen'/>
   </devices>
diff --git a/tests/xmconfigdata/test-no-source-cdrom.xml b/tests/xmconfigdata/test-no-source-cdrom.xml
index d4c2fd6..246123c 100644
--- a/tests/xmconfigdata/test-no-source-cdrom.xml
+++ b/tests/xmconfigdata/test-no-source-cdrom.xml
@@ -49,7 +49,7 @@
     <input type='keyboard' bus='ps2'/>
     <graphics type='vnc' port='-1' autoport='yes'/>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='16384' heads='1' primary='yes'/>
     </video>
     <memballoon model='xen'/>
   </devices>
diff --git a/tests/xmconfigdata/test-paravirt-net-e1000.xml b/tests/xmconfigdata/test-paravirt-net-e1000.xml
index ad44d11..735e8b0 100644
--- a/tests/xmconfigdata/test-paravirt-net-e1000.xml
+++ b/tests/xmconfigdata/test-paravirt-net-e1000.xml
@@ -33,7 +33,7 @@
       <listen type='address' address='127.0.0.1'/>
     </graphics>
     <video>
-      <model type='xen' heads='1' primary='yes'/>
+      <model type='xen' vram='4096' heads='1' primary='yes'/>
     </video>
     <memballoon model='xen'/>
   </devices>
diff --git a/tests/xmconfigdata/test-paravirt-net-vifname.xml b/tests/xmconfigdata/test-paravirt-net-vifname.xml
index e94b685..3ce3639 100644
--- a/tests/xmconfigdata/test-paravirt-net-vifname.xml
+++ b/tests/xmconfigdata/test-paravirt-net-vifname.xml
@@ -34,7 +34,7 @@
       <listen type='address' address='127.0.0.1'/>
     </graphics>
     <video>
-      <model type='xen' heads='1' primary='yes'/>
+      <model type='xen' vram='4096' heads='1' primary='yes'/>
     </video>
     <memballoon model='xen'/>
   </devices>
diff --git a/tests/xmconfigdata/test-paravirt-new-pvfb-vncdisplay.xml b/tests/xmconfigdata/test-paravirt-new-pvfb-vncdisplay.xml
index 3ca3023..20c31ec 100644
--- a/tests/xmconfigdata/test-paravirt-new-pvfb-vncdisplay.xml
+++ b/tests/xmconfigdata/test-paravirt-new-pvfb-vncdisplay.xml
@@ -32,7 +32,7 @@
       <listen type='address' address='127.0.0.1'/>
     </graphics>
     <video>
-      <model type='xen' heads='1' primary='yes'/>
+      <model type='xen' vram='4096' heads='1' primary='yes'/>
     </video>
     <memballoon model='xen'/>
   </devices>
diff --git a/tests/xmconfigdata/test-paravirt-new-pvfb.xml b/tests/xmconfigdata/test-paravirt-new-pvfb.xml
index d97799a..33c48f4 100644
--- a/tests/xmconfigdata/test-paravirt-new-pvfb.xml
+++ b/tests/xmconfigdata/test-paravirt-new-pvfb.xml
@@ -32,7 +32,7 @@
       <listen type='address' address='127.0.0.1'/>
     </graphics>
     <video>
-      <model type='xen' heads='1' primary='yes'/>
+      <model type='xen' vram='4096' heads='1' primary='yes'/>
     </video>
     <memballoon model='xen'/>
   </devices>
diff --git a/tests/xmconfigdata/test-pci-devs.xml b/tests/xmconfigdata/test-pci-devs.xml
index 4c3f5f2..a88853d 100644
--- a/tests/xmconfigdata/test-pci-devs.xml
+++ b/tests/xmconfigdata/test-pci-devs.xml
@@ -49,7 +49,7 @@
     <input type='keyboard' bus='ps2'/>
     <graphics type='vnc' port='-1' autoport='yes'/>
     <video>
-      <model type='cirrus' heads='1' primary='yes'/>
+      <model type='cirrus' vram='16384' heads='1' primary='yes'/>
     </video>
     <hostdev mode='subsystem' type='pci' managed='no'>
       <source>
